iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -955,4 +1046,44 @@ private void validateBasicStats(Table icebergTable, String dbName, String tableN
       Assert.assertEquals(summary.get(entry.getValue()), hmsParams.get(entry.getKey()));
     }
   }
+
+  private void validateMigration(String tableName, int recordCount) {
+    List<Object[]> originalResult = shell.executeStatement("SELECT * FROM " + tableName + " ORDER BY a");
+    Assert.assertEquals(recordCount, originalResult.size());
+    List<Object[]> originalDescribe = shell.executeStatement("DESCRIBE FORMATTED " + tableName);
+    validateDescribeOutput(originalDescribe, fileFormat.name());
```

Comment:
I'm not sure we need this, since this is only checking hive behaviour, not iceberg

Reply:
You are absolutely right.